cmake/EthPolicy.cmake

```diff
@@ -7,10 +7,6 @@ macro (eth_policy)
 	# let cmake autolink dependencies on windows
 	cmake_policy(SET CMP0020 NEW)
 
-	# CMake 2.8.12 and lower allowed the use of targets and files with double
-	# colons in target_link_libraries,
-	cmake_policy(SET CMP0028 OLD)
```

Comment:
Is this not used / not a problem anymore? What is the rationale keeping / removing it?

Reply:
The CMake was complaining about it. If you have `cmake_minimum_required(VERSION 3.5)` it sets the policies according to this version so this becomes redundant and is not allowed any more. 

From the recent CMake:

> The OLD behavior of a policy is deprecated by definition and may be removed in a future version of CMake.

Reply:
Oh right, I have the same warning for CMP0042 and CMP0043.